sound/sound-font-config.ini-configuration.md

```diff
@@ -139,6 +139,19 @@ ProffieOSOnImageDuration=5000.0
 # melodies which would be played in sync with the beat.
 ProffieOSSmoothSwingHumstart=0
 
+### ---- OS 8.x and above ----
+
+# Smoothswings default to switch back to the other sound again after 180 degrees.
+# This 180 degrees is controllable by using the following:
+
+#Low2HighSeparationDegrees=180
+
+#High2LowSeparationDegrees=180
+
+# How long it takes for the idle sound to fade in when
```

Comment:
Might be nice to mention that it requires a define.
I will add this for now though.